config/samples/perf_v1alpha1_pgbench.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: perf.kubestone.xridge.io/v1alpha1
+kind: Pgbench
+metadata:
+  name: pgbench-sample
+spec:
+  image:
+    name: postgres:11.5-alpine
```

Comment:
we should use `xridge/pgbench` image here

Reply:
done